pkg/apis/migration/v1alpha1/migtoken_types.go

```diff
@@ -75,6 +75,31 @@ func (r *MigToken) CanI(client k8sclient.Client, namespace, resource, verb strin
 	return sar.Status.Allowed, nil
 }
 
+// Check if the user has `use` verb on the associated MigCluster
+func (r *MigToken) HasUseVerb(client k8sclient.Client) (bool, error) {
```

Comment:
in order for this to be consistent with other utility function would you consider renaming this to `HasUsePermission`?